Explorer/Assets/DCL/Communities/CommunitiesCard/Members/MembersListController.cs

```diff
@@ -267,12 +279,21 @@ async UniTaskVoid RemoveModeratorAsync(CancellationToken token)
 
         private void CallUser(MemberData profile)
         {
+            //TODO: call user in private conversation
             throw new NotImplementedException();
         }
 
-        private void OpenChatWithUser(MemberData profile)
+        private async void OpenChatWithUser(MemberData profile)
```

Comment:
Shouldn't you rename this method to `OpenChatWithUserAsync`?
```suggestion
        private async void OpenChatWithUserAsync(MemberData profile)
```